[HOTFIX] Optimize presto-guide (#3065)
apache/carbondata: docs/presto-guide.md

To run it in foreground.

### Start Presto CLI

> 

Author: yes, removed "./presto".
The next script is for start presto server:
./presto --server <coordinator_ip>:8086 --catalog carbondata --schema <schema_name>

  Secondly: Create a folder named 'carbondata' under $PRESTO_HOME$/plugin and
  copy all jars from carbondata/integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT
        to $PRESTO_HOME$/plugin/carbondata

>   **NOTE:**  Not copy one assemble jar, need to copy many jars from integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT

Reviewer: ```suggestion
  **NOTE:**  Don't just copy assemble jar, make sure to copy all jars from integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT
```

Author: How about : Not copy the assemble jar, make sure to copy all jars ...

Reviewer: @chenliang613  "Not copy" is oral， suggest  “No need to copy assemble jar” or “”assemble jar is not required“”

Reviewer 2: Why is this NOTE needed? I think line#221 has already made it clear.

Reviewer 3: Is it the folder name can not draw users' attention? I found that a ZIP file is created also. What about asking user to extract the jars in the zip file to a folder under presto

Author: > Why is this NOTE needed? I think line#221 has already made it clear.

Many community guys made mistakes, this is why add one more note.
